sdks/python/apache_beam/io/gcp/healthcare/dicomio_integration_test.py

```diff
@@ -40,9 +40,9 @@
 
 # pylint: disable=wrong-import-order, wrong-import-position
 try:
-  from apache_beam.io.gcp.dicomclient import DicomApiHttpClient
-  from apache_beam.io.gcp.dicomio import DicomSearch
-  from apache_beam.io.gcp.dicomio import UploadToDicomStore
+  from apache_beam.io.gcp.healthcare.dicomclient import DicomApiHttpClient
```

Comment:
Have you tested locally that these tests also pass without any changes?

Reply:
Yes

<img width="1867" alt="Screenshot 2023-06-02 at 9 45 35 AM" src="https://github.com/apache/beam/assets/26037657/961c2491-6e48-4e69-865b-32715e8fb54a">
<img width="1875" alt="Screenshot 2023-06-02 at 9 45 48 AM" src="https://github.com/apache/beam/assets/26037657/4d065dcb-eb0c-4b13-9c36-b9af266031ca">